packages/modal/src/index.ts

```diff
@@ -4,3 +4,4 @@ export * from "./components/center-modal/CenterModal";
 export * from "./components/loading-modal/LoadingModal";
 export * from "./components/modal/Modal";
 export * from "./components/modal/ModalHeader";
+export * from "./components/modal/ModalFooter";
```

Comment:
Do not export `ModalFooter`, let that be an internal component that the user doesn't have to know about.

Reply:
Fixed!